src/main/resources/db/changelog/changelog-1.0.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet id="changelog-1.0" author="foxrev">
+        <createTable tableName="scooter">
+            <column name="id" type="uuid">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="serial_number" type="text">
```

Comment:
shouldn't the serial number be unique?

Reply:
If scooters have different trademarks - may  by not unique. In our app, maybe just uuid is unique identifier for scooter